assets/app/view/game/part/location_name.rb

```diff
@@ -11,6 +11,10 @@ class LocationName < Base
         BACKGROUND_COLOR = '#FFFFFF'
         BACKGROUND_OPACITY = '0.5'
 
+        def edge_at_bottom?(edge = nil)
```

Comment:
why is edge default nil? i don't see you calling it without args

in one case would it be nil? for the safety check on line 15?

Reply:
Just being paranoid. I'll delete.